src/helpers.nim

```diff
@@ -37,6 +37,27 @@ proc setFalseAndPrint*(b: var bool; description: string; path: Path) =
   stdout.writeLine(path.string)
   stdout.write "\n"
 
+var printedWarning* = false
```

Comment:
This global is bad style. We should refactor it at some point, but it's okay for now.